docs/UserGuide.md

```diff
@@ -213,15 +215,15 @@ Deletes the specific passenger from the GME terminal.
 
 Selects passengers from the current view in the bottom left pane of the GME terminal to arrange a carpool. 
 
-**Format:** `pool n/DRIVER_NAME p/DRIVER_PHONE d/TRIPDAY t/TRIPTIME c/INDEX [c/INDEX c/INDEX ...] [tag/TAG]`
+**Format:** `pool n/DRIVER_NAME p/DRIVER_PHONE d/TRIPDAY t/TRIPTIME c/INDEX [c/OPTIONAL_INDEX c/OPTIONAL_INDEX ...] [tag/TAG]`
 
 <div markdown="block" class="alert alert-info">
 
 **:information_source: Notes about the pool command:**<br>
 
 * [TripDay](#6-glossary) is required to be a valid day of the week. e.g. `SUNDAY` or `FRIDAY`.
 * [TripTime](#6-glossary) is required to be in the 24-hour format. e.g. `0530` or `2359`.
-* Here, c/ stands for commuter, i.e. someone to be pooled with a driver.
+* Here, `c/` stands for commuter, i.e. someone to be pooled with a driver.
```

Comment:
according to our 'navigating the user guide', the code formatting is reserved for commands that can be executed as well, technically a prefix isn't a command but i get that it's part of the command also